README.md

```diff
@@ -19,4 +19,23 @@ and interesting experience to using skin widgets.
     
 6.  Return to "Add-ons", and choose "Install from repository".
 
-7.  Choose the repository you installed (stable or dev), and install AutoWidget from Program Add-ons.
\ No newline at end of file
+7.  Choose the repository you installed (stable or dev), and install AutoWidget from Program Add-ons.
+
+## Steps to Use
+1. Open any other plugin you wish and browse to a menu item for a list you want
+2. On the context menu select "Add to autowidget group".
+3. Either create a new group or add to an existing group. Groups can have more than one list or "path" inside it
+4. Inside your skin add a new widget
+5. For your widget Select Add ons > Autowidgets > My Groups > {your group name} > {group mode}

```

Comment:
This path doesn't seem generally correct. You usually need to choose "Video Add-ons", etc... in between. Also, "Autowidgets".